frontend/apps/main/src/components/editor/editorStyles.scss

```
@@ -0,0 +1,199 @@
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.tiptap.ProseMirror {
+  flex: 1;
+  min-height: 70px;
+  overflow-y: auto;
+  word-wrap: break-word !important;
+  overflow-wrap: break-word !important;
+  word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)